### PR TITLE
feat: Github actions workflow

### PR DESCRIPTION
**Pull request by Aiky30**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ 3.7, 3.8, 3.9 ]  # latest release minus two
+        requirements-file: [
+            dj22_cms40.txt,
```
Comment (Aiky30) [inline .github/workflows/test.yml:13]:
```suggestion
            django_22.txt,
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ 3.7, 3.8, 3.9 ]  # latest release minus two
+        requirements-file: [
+            django_22.txt,
+            dj32_cms40.txt,
```
Comment (Aiky30) [inline .github/workflows/test.yml:14]:
```suggestion
            django_32.txt,
```

**Review by NarenderRajuB — COMMENTED** [review]
LGTM

**Review by NarenderRajuB — APPROVED** [review]

**Comment by Aiky30** [comment]
Closing as GH actions are now firing.

**Pull request closed without merging**